### PR TITLE
Update docs for timeseries integration options

### PR DESCRIPTION
**Pull request by mmahacek**
### All Contributors

* [X] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [X] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### External References

* JIRA (Issue Tracker): No Jira

### CODE REVIEW

**Review by Bonrob2 — DISMISSED** [review]
LGTM!

File: docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc [hunk 1]
```diff
@@ -57,7 +57,7 @@ Must be a power of two.
 | -1 (override by `org.opennms.timeseries.config.ring_buffer_size`)
 
 | org.opennms.timeseries.config.writer_threads
-| Number of threads used to pull samples from the ring buffer and insert them into the time-series database.
+| Number of threads used to pull samples from the buffer and insert them into the time-series database.
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc:60]: We should standardize how we refer to the database type. In some docs, we have "time series database," while in others (like here) we have "time-series database."

File: docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc [hunk 2]
```diff
@@ -153,9 +153,9 @@ You can tune the cache settings to suit your needs, using the parameters above.
 | TimeseriesPersisterMetaTagCache  | Caches all additionally configured and resolved meta-tag values by resource.
 |===
 
-The samples the collectors gather are temporarily stored in a ring buffer before they are persisted to the Time Series Integration Layer.
-Increase the value of the ring_buffer_size if you expect large peaks of collectors returning at once or latency in persisting these.
-However, note that the memory the ring buffer uses is reserved, and larger values may require an increased heap size.
+The samples the collectors gather are temporarily stored in a buffer before they are persisted to the Time Series Integration Layer.
+Increase the value of the buffer_size if you expect large peaks of collectors returning at once or latency in persisting these.
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc:157]:
```suggestion
Increase the value of the `buffer_size` if you expect large peaks of collectors returning at once or latency in persisting these.
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc [hunk 3]
```diff
@@ -169,18 +169,20 @@ The configuration of the tags has the following form:
 * The prefix is: `org.opennms.timeseries.tin.metatags.tag`
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc:169]:
```suggestion
* The prefix is `org.opennms.timeseries.tin.metatags.tag`.
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc [hunk 4]
```diff
@@ -169,18 +169,20 @@ The configuration of the tags has the following form:
...
 * The tagKey can be an arbitrary string as long as it does not break the Java properties file https://en.wikipedia.org/wiki/.properties[syntax].
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc:170]:
```suggestion
* The `tagKey` can be an arbitrary string, as long as it does not break the https://en.wikipedia.org/wiki/.properties[Java properties file syntax].
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc [hunk 5]
```diff
@@ -169,18 +169,20 @@ The configuration of the tags has the following form:
...
 * The query expression lets you query the value.
-Supports use of the xref:operation:deep-dive/meta-data.adoc#ga-meta-data-dsl[Metadata DSL].
+Supports use of the xref:operation:deep-dive/meta-data.adoc#ga-meta-data-dsl[Metadata DSL] within the node, asset, and requisition contexts.
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc:172]:
```suggestion
It supports use of the xref:operation:deep-dive/meta-data.adoc#ga-meta-data-dsl[Metadata DSL] within the node, asset, and requisition contexts.
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc [hunk 6]
```diff
@@ -169,18 +169,20 @@ The configuration of the tags has the following form:
...
 
 Examples:
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc:174]:
```suggestion
The following examples demonstrate proper tag definition syntax:
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc [hunk 7]
```diff
@@ -153,9 +153,9 @@ You can tune the cache settings to suit your needs, using the parameters above.
...
 
-The samples the collectors gather are temporarily stored in a ring buffer before they are persisted to the Time Series Integration Layer.
-Increase the value of the ring_buffer_size if you expect large peaks of collectors returning at once or latency in persisting these.
-However, note that the memory the ring buffer uses is reserved, and larger values may require an increased heap size.
+The samples the collectors gather are temporarily stored in a buffer before they are persisted to the Time Series Integration Layer.
+Increase the value of the buffer_size if you expect large peaks of collectors returning at once or latency in persisting these.
+However, note that the memory the buffer uses is reserved, and larger values may require an increased heap size.
 
 [[ga-opennms-operation-timeseries-properties-meta-tags]]
 == Expose additional meta tags
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc:161]:
```suggestion
== Expose additional meta tags

```

File: docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc [hunk 8]
```diff
@@ -169,18 +169,20 @@ The configuration of the tags has the following form:
...
 * `org.opennms.timeseries.tin.metatags.tag.sysObjectID=${node:sys-object-id}`
 
-Expose categories by setting `org.opennms.timeseries.tin.metatags.exposeCategories` to `true`.
+=== Expose categories as meta tags
+
+You can expose node categories as meta tags by setting `org.opennms.timeseries.tin.metatags.exposeCategories` to `true`.
 
 * `org.opennms.timeseries.tin.metatags.exposeCategories=true` +
 
 Will lead to:
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc:185]:
```suggestion
This results in the following tags:
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc (hunk 1 above)
Comment (Bonrob2) [reply docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc:60]: Not hyphenated seems to be standard use, so we should not hyphenate it. Good catch!

Comment (mmahacek) [reply docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc:60]:
```suggestion
| Number of threads used to pull samples from the buffer and insert them into the time series database.
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc [hunk 9]
```diff
@@ -153,34 +153,34 @@ You can tune the cache settings to suit your needs, using the parameters above.
 | TimeseriesPersisterMetaTagCache  | Caches all additionally configured and resolved meta-tag values by resource.
 |===
 
-The samples the collectors gather are temporarily stored in a ring buffer before they are persisted to the Time Series Integration Layer.
-Increase the value of the ring_buffer_size if you expect large peaks of collectors returning at once or latency in persisting these.
-However, note that the memory the ring buffer uses is reserved, and larger values may require an increased heap size.
+The samples the collectors gather are temporarily stored in a buffer before they are persisted to the Time Series Integration Layer.
+Increase the value of the `buffer_size` if you expect large peaks of collectors returning at once or latency in persisting these.
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc:157]:
```suggestion
You should increase the value of the `buffer_size` if you expect large peaks of collectors returning at once or latency in persisting these.
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 10]
```diff
@@ -1,60 +1,112 @@
...
 ----
+--
+====
 
-=== Obtain an activation key
+NOTE: The plugin may be updated separately from {page-component-title} releases.
+The above steps can be used on Horizon 31 and later to ensure you have the latest version of the plugin.
 
-The activation key lets {page-component-title} connect with OpenNMS cloud services (currently the Time Series DB).
+To ensure that the feature is installed on service restarts, add the feature to a file in `featuresBoot.d`:
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:54]:
```suggestion
To ensure that the feature is installed on service restarts, add the feature to a file in `featuresBoot.d`:

```

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 11]
```diff
@@ -1,60 +1,112 @@
...
 . Choose *OpenNMS Core Instances* in the left menu and click *Add New*.
 . Type in the name of your instance and click *Generate Activation Key*.
 . Copy the activation key and click *Save & Close*.
 
-+
-NOTE: The key is valid for one hour.
-If you do not use it within that time, you must log back into the OpenNMS Portal, delete the old instance, create a new one, and generate a new key.
+=== Enable cloud services connectivity
+
+To install the activation key on your {page-component-title} instance:
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:109]:
```suggestion
To install the activation key on your {page-component-title} instance, follow these steps:
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 12]
```diff
@@ -1,60 +1,113 @@
...
 
 == Requirements
 
 You must have the following to use the Time Series DB:
 
-* Horizon 31+ (Meridian availability with the 2023 release).
-* Time Series DB subscription that provides access to the https://portal.opennms.com[OpenNMS Portal] to retrieve your activation key and view Time Series DB metrics.
-
+* Horizon 30+ (Meridian availability with the 2023 release).
+* Time Series DB subscription, which provides access to the https://portal.opennms.com[OpenNMS Portal] to retrieve your activation key and view Time Series DB metrics.
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:11]:
```suggestion
* A Time Series DB subscription, which provides access to the https://portal.opennms.com[OpenNMS Portal] to retrieve your activation key and view Time Series DB metrics.
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 13]
```diff
@@ -1,60 +1,113 @@
...
 
 == Deployment
 
-There are three phases of deployment to set up the Time Series DB for your system:
+There are four steps to set up the Time Series DB for your system:
 
 * Update OpenNMS Cloud Services Connector files.
+* Configure plugin settings.
 * Obtain an activation key for the Cloud Services Connector.
 * Enable cloud services connectivity.
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:21]:
```suggestion
. Update OpenNMS Cloud Services Connector files.
. Configure plugin settings.
. Obtain an activation key for the Cloud Services Connector.
. Enable cloud services connectivity.
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 14]
```diff
@@ -1,60 +1,113 @@
...
 
 === Update OpenNMS Cloud Services Connector files
 
-The OpenNMS Cloud Services Connector is an extension that lets a {page-component-title} instance send time series data to an OpenNMS-hosted cloud service.
+The OpenNMS Cloud Services Connector is an extension that lets a {page-component-title} instance send time series data to the OpenNMS-hosted cloud service.
+The packages for this extension are installed by default in Horizon 31 and later, but do need to be configured and activated.
 
-This extension is installed by default in Horizon 31.
-However, you need to move the configuration files into the correct location and restart {page-component-title} as part of the Time Series DB setup process:
+For Horizon 30, the plugin can be installed from our package repositories.
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:28]:
```suggestion
For Horizon 30, the plugin can be installed from our package repositories:
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 15]
```diff
@@ -1,60 +1,113 @@
...
+[source, console]
+----
+echo "opennms-plugin-cloud-core wait-for-kar=opennms-plugin-cloud" | sudo tee ${OPENNMS_HOME}/etc/featuresBoot.d/plugin-cloud.boot
+----
+
+=== Configure OpenNMS Cloud Services settings
 
+You will need to add settings to tell your {page-component-title} instance to load the plugin on service start and to direct storage to the cloud service.
+
+.Horizon 31 and later
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:65]:
```suggestion
If you are running Horizon 31 or later, copy the example `timeseries.properties` file to your working directory:

```

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 16]
```diff
@@ -1,60 +1,113 @@
...
+[source, console]
+----
+cp ${OPENNMS_HOME}/etc/examples/opennms.properties.d/timeseries.properties ${OPENNMS_HOME}/etc/opennms.properties.d/timeseries.properties
+----
+
+.On Horizon 30, create `$\{OPENNMS_HOME}/etc/opennms.properties.d/timeseries.properties`
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:71]:
```suggestion
If you are running Horizon 30, create `$\{OPENNMS_HOME}/etc/opennms.properties.d/timeseries.properties` and populate it with the following settings:

```

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 17]
```diff
@@ -1,60 +1,113 @@
...
+org.opennms.timeseries.strategy=integration
+org.opennms.timeseries.config.buffer_type=OFFHEAP
+org.opennms.timeseries.config.writer_threads=16
+org.opennms.timeseries.config.buffer_size=131072
+org.opennms.timeseries.config.offheap.batch_size=8192
+org.opennms.timeseries.config.offheap.path=/tmp
+org.opennms.timeseries.config.offheap.max_file_size=-1
+----
+
+After the plugin is installed and configured, restart the {page-component-title} service.
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:83]:
```suggestion
After the plugin is installed and configured, restart the {page-component-title} service:
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 18]
```diff
@@ -1,60 +1,113 @@
...
+[source, console]
+----
+systemctl restart opennms
+----
+
+=== Obtain an activation key
+
+The activation key lets {page-component-title} connect with the Time Series DB cloud service.
+You must log in to the OpenNMS Portal to get this activation key, copy it, and then paste it into the appropriate screen in {page-component-title}.
 When you signed up for the Time Series DB, you received an activation email from OpenNMS with a login for the OpenNMS Portal.
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:94]:
```suggestion

When you signed up for the Time Series DB, you received an activation email from OpenNMS with a login for the OpenNMS Portal.
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 19]
```diff
@@ -1,60 +1,113 @@
...
+If you did not receive an activation email, contact your account manager.
+
+IMPORTANT: The key is valid for one hour.
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:97]:
```suggestion
IMPORTANT: The activation key is valid for one hour.
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 20]
```diff
@@ -1,60 +1,113 @@
...
+If you do not use it within that time, you must log back into the OpenNMS Portal, delete the instance, create a new one, and generate a new key.
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:98]:
```suggestion
If you do not use it within that time, you must log in to the OpenNMS Portal again, delete the instance, create a new instance, and generate a new key.
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 21]
```diff
@@ -1,60 +1,113 @@
...
 
 To obtain the activation key, follow these steps:
 
 . If you have not already done so, activate your OpenNMS Portal account by clicking the *Account Activation* button in your welcome email.
-. On the Create Your OpenNMS Account screen, type a password in the *Password* box and click *Create My Account*.
 . Sign in to the OpenNMS Portal with your username and password.
 . Choose *OpenNMS Core Instances* in the left menu and click *Add New*.
 . Type in the name of your instance and click *Generate Activation Key*.
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:105]:
```suggestion
. Type the name of your instance and click *Generate Activation Key*.
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 22]
```diff
@@ -1,60 +1,113 @@
...
 . Copy the activation key and click *Save & Close*.
 
-+
-NOTE: The key is valid for one hour.
-If you do not use it within that time, you must log back into the OpenNMS Portal, delete the old instance, create a new one, and generate a new key.
+=== Enable cloud services connectivity
+
+To install the activation key on your {page-component-title} instance, follow these steps:
 
 . Log in to the OpenNMS {page-component-title} web UI for your instance.
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:112]:
```suggestion
. Log in to {page-component-title}.
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 23]
```diff
@@ -86,7 +139,7 @@ Connecting to ElasticSearch ReST API (Flows)   [ Success  ] => Not configured
 Check Karaf logs if the registration fails or the health check does not say `SERVING`.
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:139]:
```suggestion
Check the Karaf logs if the registration fails or the health check does not say `SERVING`.
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 24]
```diff
@@ -86,7 +139,7 @@ Connecting to ElasticSearch ReST API (Flows)   [ Success  ] => Not configured
...
 If this is the case, you will need to open a support ticket.
 
-You can also check that `OPENNMS_HOME/share/rrd` files are still being updated.
+You can also check that `$\{OPENNMS_HOME}/share/rrd` files are still being updated.
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:142]:
```suggestion
You can also check if the `$\{OPENNMS_HOME}/share/rrd` files are still being updated.
```

**Review by emarsh1995 — CHANGES_REQUESTED** [review]

**Review by emarsh1995 — APPROVED** [review]
LGTM! Thank you!

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 25]
```diff
@@ -83,10 +139,10 @@ Connecting to ElasticSearch ReST API (Flows)   [ Success  ] => Not configured
 => Everything is awesome
 ----
 
-Check Karaf logs if the registration fails or the health check does not say `SERVING`.
+Check the Karaf logs if the registration fails or the health check does not say `SERVING`.
 If this is the case, you will need to open a support ticket.
 
-You can also check that `OPENNMS_HOME/share/rrd` files are still being updated.
+You can also check if the `$\{OPENNMS_HOME}/share/rrd` files are still being updated.
 If they are not, this indicates that the Time Series DB is not working.
```
Comment (mmahacek) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:146]: @drewym @jeffgdotorg Can either of you confirm this? It seems like the opposite should be true.  If there are NO new updates to the RRD files, then the integration plugin would be handling metric storage.

Comment (mmahacek) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:146]:
```suggestion
If they are not, this indicates that the Time Series DB is working.
```

**Review by Bonrob2 — COMMENTED** [review]
This looks good. Could we wait to merge until after the holiday so we can confirm with Jeff and Andy about support for the plugin with 30. I didn't document, because my understanding was that while they can install it with 30, we weren't necessarily encouraging people to do so.

**Comment by mmahacek** [comment]
@Bonrob2 @drewym I'd like to get these changes into 31.0.2 next week.

**Comment by Bonrob2** [comment]
> @Bonrob2 @drewym I'd like to get these changes into 31.0.2 next week.

Content looks good to me, just the outstanding question of support for plugin with 30.

**Comment by Bonrob2** [comment]
> Doesn't look like we've gotten confirmation on this.

I'll ping Jeff to get his thoughts.

**Comment by Bonrob2** [comment]
@mmahacek Jeff said we do not support using the plugin with H30, so we shouldn't document it.

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 26]
```diff
@@ -1,62 +1,118 @@
...
+The OpenNMS Time Series DB is a cloud-hosted service that provides a scalable time series database to store and persist performance metrics that {page-component-title} collects from the devices it monitors.
 
 == Requirements
 
 You must have the following to use the Time Series DB:
 
-* Horizon 31+ (Meridian availability with the 2023 release).
-* Time Series DB subscription that provides access to the https://portal.opennms.com[OpenNMS Portal] to retrieve your activation key and view Time Series DB metrics.
-
+* Horizon 30+ (Meridian availability with the 2023 release).
```
Comment (mmahacek) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:10]:
```suggestion
* Horizon 31+ (Meridian availability with the 2023 release).
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 27]
```diff
@@ -1,62 +1,118 @@
...
+[source, console]
+----
+sudo apt install opennms-cloud-plugin
 ----
+--
+====
 
-=== Obtain an activation key
+NOTE: The plugin may be updated separately from {page-component-title} releases.
+The above steps can be used on Horizon 31 and later to ensure you have the latest version of the plugin.
```
Comment (mmahacek) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:52]:
```suggestion
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 28]
```diff
@@ -1,62 +1,118 @@
...
+
+To ensure that the feature is installed on service restarts, add the feature to a file in `featuresBoot.d`:
```
Comment (mmahacek) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:54]:
```suggestion
To ensure that the feature is loaded on service restarts, add the feature to a file in `featuresBoot.d`:
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 29]
```diff
@@ -1,62 +1,118 @@
...
+----
+org.opennms.timeseries.strategy=integration
+org.opennms.timeseries.config.buffer_type=OFFHEAP
+org.opennms.timeseries.config.writer_threads=16
+org.opennms.timeseries.config.buffer_size=131072
+org.opennms.timeseries.config.offheap.batch_size=8192
+org.opennms.timeseries.config.offheap.path=/tmp
+org.opennms.timeseries.config.offheap.max_file_size=-1
+----
+
```
Comment (mmahacek) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:84]:
```suggestion
```

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 30]
```diff
@@ -1,62 +1,79 @@
...
+. Update OpenNMS Cloud Services Connector files.
+. Configure plugin settings.
+. Obtain an activation key for the Cloud Services Connector.
+. Enable cloud services connectivity.
 
 === Update OpenNMS Cloud Services Connector files
 
-The OpenNMS Cloud Services Connector is an extension that lets a {page-component-title} instance send time series data to an OpenNMS-hosted cloud service.
+The OpenNMS Cloud Services Connector is an extension that lets a {page-component-title} instance send time series data to the OpenNMS-hosted cloud service.
+The packages for this extension are installed by default in Horizon 31 and later, but do need to be configured and activated.
```
Comment (mmahacek) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:27]:
```suggestion
You can use `yum` or `apt` to make sure the `opennms-cloud-plugin` package is up to date with the most current version of the plugin.

```

**Review by Bonrob2 — APPROVED** [review]
LGTM! Thanks!

**Review by Bonrob2 — APPROVED** [review]
LGTM!

**Pull request merged**